README.md

```diff
@@ -354,6 +354,37 @@ Checkout the new branch to continue working:
 (master)$ git checkout my-branch
 ```
 
+<a name="keep-whole-file"></a>
+### I want to keep the whole file from another ref-ish
+
+Say you have a working spike, with hundreds of changes. Everything is working. Now, you commit into another branch to save that work.
```

Comment:
Ok. Can you add this:

```sh
(solution)$ git add -A && git commit -m "Adding all changes from this spike into one big commit."
```

This will help readability, I think.